0.5.0 release (#97)
packit-service/ogr: CHANGELOG.md

# 0.5.0

## New Features

* Add support for Github issues. (@marusinm)
* New methods for updating pull-requests. (@lbarcziova)
* New methods for getting forks for user/project. (@lachmanfrantisek)

## Fixes

* Better support for forks and forking. (@lachmanfrantisek)

> * Fix problem when Pagure token is not set. (@lachmanfrantisek)

Reviewer: ```suggestion
* Fix a problem when Pagure token is not set. (@lachmanfrantisek)
```

## Minor

> * Write mode in testing is determined form the file (non)existence. (@lachmanfrantisek)

Reviewer: ```suggestion
* Write mode in testing is determined whether a respective offline file exists or not. (@lachmanfrantisek)
```
